gateway-spi/src/main/java/org/apache/knox/gateway/config/GatewayConfig.java

```diff
@@ -379,6 +379,13 @@
    */
   int getWebsocketIdleTimeout();
 
+  /**
+   * Max count of messages that can be temporarily buffered in memory before a connection is properly setup.
+   * @since 0.10
+   * @return buffer size
+   */
+  int getWebsocketMaxWaitBufferCount();
```

Comment:
See my previous comments about this method.

Reply:
Fixed.

Reply:
Thanks!